Week1/homework/js-exercises/logCompare.js

```diff
@@ -0,0 +1,15 @@
+'use script';
+
+{
+  const myArr1 = ['n', 'u', 'r', 'i'];
+  const myArr2 = [' ', 'b', 'a', 'y', 'r', 'a', 'm'];
+
+  let x = console.log(myArr1.length);
+  let y = console.log(myArr2.length);
+
+  if ((x = y)) {
```

Comment:
Aahaan. There is a mistake here. Please have a careful look. 🙂 

Also, we can use `const` instead of `let` while creating `x` and `y` since we needn't re-assign them.